### PR TITLE
:sparkles: Prepare Detail Page

### PR DESCRIPTION
**Pull request by welschoff**
Screenshot 2021-12-01 at 10 34 44

### CODE REVIEW

File: src/pages/Detail/Detail.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function Detail() {
+  const [series, setSeries] = useState<DetailCardProps | null>(null);
+
+  const getDetails = async () => {
+    const response = await fetch('http://localhost:3001/api/details');
+    const data = await response.json();
+    setSeries(data);
+  };
+
+  // console.log(series?.genres[0].name);
```
Comment (PReichetanz) [inline src/pages/Detail/Detail.tsx:17]:
```suggestion
```

**Review by PReichetanz — COMMENTED** [review]
Looks good to me!

File: src/pages/Detail/Detail.tsx (hunk 1 above)
Comment (lajanzen) [inline src/pages/Detail/Detail.tsx:17]:
```suggestion
```

**Review by lajanzen — COMMENTED** [review]
In your deployment the API results on your details page aren't shown. I think this might be fixed when you add the proxy to your project like Leon explained this morning :)

File: src/pages/Detail/Detail.tsx (hunk 1 above)
Comment (welschoff) [reply src/pages/Detail/Detail.tsx:17]: Maybe I want to use this later, so I don't want to delete it completely..

**Pull request merged**